src/utils/PerfherderGraphContainer.jsx

```diff
@@ -176,11 +200,9 @@ const perfherderFormatter = (series, timeDomain) => {
       ...row,
       // choose meta from the source with most recent data
       meta: selectFrom(sources)
-        .sort(s =>
-          selectFrom(s.data)
-            .select('push_timestamp')
-            .max()
-        )
+        .sort(s => selectFrom(s.data)
+          .select('push_timestamp')
+          .max())
```

Comment:
ensuring the opening parenthesis and closing parenthesis do not line up is a mistake

Reply:
We can fix that.

Reply:
We can?!

Reply:
I haven't tried. Maybe.